Cargo.toml

```diff
@@ -126,15 +131,15 @@ awc = { path = "awc" }
 
 [[test]]
 name = "test_server"
-required-features = ["compress", "cookies"]
+required-features = ["compress-all", "cookies"]
```

Comment:
Does it really require `compress-all` though?

Reply:
Actually not all example / tests requires all compress algorithm. But this one does.
I just review required feature for tests / example. Bellow are the changes:
```toml
[[test]]
name = "test_server"
# Require all compression features because there is tests for Brotli + Gzip + Zstd in test/test_server.rs
required-features = ["compress-brotli", "compress-gzip", "compress-zstd", "cookies"]

[[example]]
name = "basic"
# Require any compression flag because it use middleware::Compress
required-features = ["compress-gzip"]

[[example]]
name = "uds"
# Require any compression flag because it use middleware::Compress
required-features = ["compress-gzip"]
```